src/web/app/pages-admin/admin-notifications-page/notification-edit-form/notification-edit-form-datepicker-formatter.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import { NgbDateParserFormatter, NgbDateStruct } from '@ng-bootstrap/ng-bootstrap';
+import moment from 'moment-timezone';
+
+const DATE_FORMAT: string = 'ddd, DD MMM, YYYY';
+
+/**
+ * Date formatter for date picker in notification edit form
+ */
+@Injectable()
+export class NotificationEditFormDatePickerFormatter extends NgbDateParserFormatter {
```

Comment:
This class seems generic enough to place it somewhere more common. e.g. components/teammates-common

Reply:
Thanks for the comment. It has been moved to be under the components/datepicker folder as this is datepicker-specific formatter.